gtsfm/evaluation/visualize_benchmark_comparison.py

```diff
@@ -226,9 +228,17 @@ def generate_dashboard(curr_master_dirpath: str, new_branch_dirpath: str) -> Non
                 else:
                     percentage_change = metrics_utils.compute_percentage_change(float(master_val), float(branch_val))
 
+                # For some metrics, smaller is better, so this will flip the color to green for reduced values, instead of red:
+                # exception are outlier errors, which we want to get larger.
                 if "error" in metric_name and "outlier" not in metric_name:
-                    # smaller is better, so this will flip the color to green for reduced values, instead of red
-                    # exception are outlier errors, which we want to get larger.
+                    percentage_change *= -1
+                elif "outlier" in metric_name and "error" not in metric_name:
+                    percentage_change *= -1
+                elif "EXCEEDS" in metric_name:
+                    percentage_change *= -1
+                elif "failure_ratio" in metric_name:
+                    percentage_change *= -1
+                elif "CHEIRALITY_FAILURE" in metric_name:
```

Comment:
Might not be straightforward as we have reprojection result w.r.t GT cams and w.r.t computed CAMS. Anyway, I am fine with it.